packages/front-end/styles/_bootstrap-theme-overrides.scss

```diff
@@ -0,0 +1,372 @@
+// Global Bootstrap overrides that respect dark/light modes
+
+// region Typography
+
+a,
+.btn-link {
+  color: var(--text-link-color);
+
+  &:hover {
+    color: var(--text-link-hover-color);
+  }
+}
+
+.text-dark,
+a.text-dark:hover,
+a.text-dark:focus {
+  color: var(--text-dark) !important;
+}
+
+.text-muted {
+  color: var(--text-color-muted) !important;
+}
+
+// endregion Typography
+
+// region Forms
+
+.form-control,
+.form-control:focus {
+  background-color: var(--surface-background-color);
+  color: var(--text-color-main);
+}
+
+.form-control {
+  border-color: var(--border-color-300);
+
+  &[readonly] {
+    background-color: var(--surface-background-color-alt);
+  }
+}
+
+.form-control:focus {
+}
+
+.input-group-text {
+  border-color: var(--border-color-300);
+  background-color: var(--form-input-group-background);
+  color: var(--form-input-group-color);
+}
+
+.form-group,
+.gb-select,
+.gb-multi-select {
+  [class$="-menu"] {
+    background-color: var(--surface-background-color);
+  }
+
+  [class$="-option"] {
+    color: var(--text-color-main);
+  }
+
+  [class$="-Input"] {
+    color: var(--text-color-main);
+  }
+
+  [class$="-singleValue"] {
+    color: var(--text-color-muted);
+  }
+
+  [class$="-control"] {
+    background-color: var(--surface-background-color);
+    border-color: var(--border-color-300);
+  }
+
+  [class$="-multiValue"] {
+    background-color: var(--form-multivalue-background-color);
+
+    > * {
+      color: var(--form-multivalue-text-color);
+    }
+  }
+}
```

Comment:
These selectors are quite fragile. If we update the library we use for multiple select, there's a good chance this will break. That said, if we do that, it's likely the developer would test that and we'd catch it, so probably ok to leave them.